specification/EnterpriseKnowledgeGraph/data-plane/Microsoft.EnterpriseKnowledgeGraph/stable/2018-12-03/EnterpriseKnowledgeGraphSwagger.json

```diff
@@ -0,0 +1,2001 @@
+            }
+          },
+          "describeVariables": {
+            "uniqueItems": false,
+            "type": "array",
+            "items": {
+              "$ref": "#/definitions/IToken"
+            }
+          },
+          "queryType": {
+            "format": "int32",
+            "description": "\r\n0=Unknown; \r\n1=Ask; \r\n2=Construct; \r\n3=Describe; \r\n4=DescribeAll; \r\n5=Select; \r\n6=SelectDistinct; \r\n7=SelectReduced; \r\n8=SelectAll; \r\n9=SelectAllDistinct; \r\n10=SelectAllReduced; \r\n",
+            "enum": [
+              0,
+              1,
+              2,
+              3,
+              4,
+              5,
+              6,
+              7,
+              8,
+              9,
+              10
+            ],
+            "type": "integer"
+          },
+          "specialType": {
+            "format": "int32",
+            "description": "\r\n0=DistinctGraphs; \r\n1=NotApplicable; \r\n2=Unknown; \r\n3=AskAnyTriples; \r\n",
+            "enum": [
+              0,
+              1,
+              2,
+              3
+            ],
+            "type": "integer"
+          },
+          "rootGraphPattern": {
+            "$ref": "#/definitions/GraphPattern"
+          },
+          "constructTemplate": {
+            "$ref": "#/definitions/GraphPattern"
+          },
+          "orderBy": {
+            "$ref": "#/definitions/ISparqlOrderBy"
+          },
+          "groupBy": {
+            "$ref": "#/definitions/ISparqlGroupBy"
+          },
+          "having": {
+            "$ref": "#/definitions/ISparqlFilter"
+          },
+          "bindings": {
+            "$ref": "#/definitions/BindingsPattern"
+          },
+          "describer": {
+            "$ref": "#/definitions/ISparqlDescribe"
+          },
+          "algebraOptimisers": {
+            "uniqueItems": false,
+            "type": "array",
+            "items": {
+              "$ref": "#/definitions/IAlgebraOptimiser"
+            }
+          },
+          "expressionFactories": {
+            "uniqueItems": false,
+            "type": "array",
+            "items": {
+              "$ref": "#/definitions/ISparqlCustomExpressionFactory"
+            }
+          },
+          "propertyFunctionFactories": {
+            "uniqueItems": false,
+            "type": "array",
+            "items": {
+              "$ref": "#/definitions/IPropertyFunctionFactory"
+            }
+          },
+          "limit": {
+            "format": "int32",
+            "type": "integer"
+          },
+          "offset": {
+            "format": "int32",
+            "type": "integer"
+          },
+          "timeout": {
+            "format": "int64",
+            "type": "integer"
+          },
+          "partialResultsOnTimeout": {
+            "type": "boolean"
+          },
+          "queryExecutionTime": {
+            "type": "string"
+          },
+          "isAggregate": {
+            "type": "boolean",
```

Comment:
This is malformed JSON. I'm not really sure why this is not being flagged by our tools. But please fix this up.